plugins/awscli.json5

```
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "description": "Bump awscli in .tool-versions",
+  "regexManagers": [
+    {
+      "fileMatch": ["\\.tool-versions$"],
+      "matchStrings": ["^awscli (?<currentValue>.+?)\\n"],
```

```suggestion
      "matchStrings": ["awscli (?<currentValue>.+?)\\n"],
```

`^` might not be worked. This is intentional try.

---

Worked 🎉 https://github.com/kachick/renovate-config-asdf/pull/195